File: reactjs/src/app/groups/group5/services/dto/jobSeekerDto/achievementDto.ts
export interface CreateAchievementInput {
    achievementName: string,
    idJobSeeker: number,
    year: number,
    organization: string,
    note: string,
    grantedPermissions: [
        string
    ]
}

export interface UpdateAchievementInput extends CreateAchievementInput {
    id: number
}

export interface GetAllAchievementByIDOutput<IAchievementItem> {
    items: IAchievementItem[]
}

File: aspnet-core/src/group5/Group5.SE347.L11_HelloWork.Application/Services/Achievements/Dto/AchievementDto.cs
using Abp.Application.Services.Dto;
using Abp.AutoMapper;
using Abp.Events.Bus.Entities;
using SE347.L11_HelloWork.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Group5.SE347.L11_HelloWork.Application.Services.Achievements.Dto
{
    [AutoMapFrom(typeof(Achievement))]
    public class AchievementDto : EntityDto<int>
    {
        public string AchievementName { get; set; }
        public long? IDJobSeeker { get; set; }
        public int Year { get; set; }
        public string Organization { get; set; }
        public string Note { get; set; }
    }
}
